src/functions.php

```diff
@@ -534,10 +506,8 @@ function parse_response($message)
  *
  * @param string   $str         Query string to parse
  * @param int|bool $urlEncoding How the query string is encoded
- *
- * @return array
  */
-function parse_query($str, $urlEncoding = true)
+function parse_query(string $str, $urlEncoding = true): array
```

Comment:
I am not sure what's the best practice to follow in cases like this..  since we are moving to PHP7

for instance if we should do a type check check and throw a TypeError in case if it is not an `int` or `bool`.. this way we are better simulating the behavior of PHP7.. but can't decide if it is an overhead or not...

Reply:
Resolving.. as there are many places that this can apply.. we can follow up with another PR.